### PR TITLE
OAS-4983 | update docs for getting started guide

### PR DESCRIPTION
**Pull request by adoi**

### CODE REVIEW

**Review by ansoboleva — CHANGES_REQUESTED** [review]

File: docs/guides/getting-started.md [hunk 1]
```diff
@@ -5,39 +5,64 @@ description: |-
 ---
 
 # Getting started with Oasis Terraform Provider
+ArangoDB Oasis, the ArangoDB Cloud, provides ArangoDB databases as a Service (DBaaS). It enables you to use the entire functionality of an ArangoDB cluster deployment without the need to run or manage the system yourself.
```
[the remark this answers is not in the document]
Comment (adoi) [reply docs/guides/getting-started.md:8]: As discussed, this is something we will decide for in the future, at the moment it is based on [ArangoDB Oasis Docs](https://www.arangodb.com/docs/stable/oasis/)

File: docs/guides/getting-started.md [hunk 2]
```diff
@@ -5,39 +5,64 @@ description: |-
...
-### Retrieving API Keys
+## Provider Setup
 
-You will need API Keys to interact with the Terraform Provider. Api keys can be generated and viewed under the user's dashboard view on the API Keys tab.
-On a logged in view, navigate to [API Keys](https://cloud.arangodb.com/dashboard/user/api-keys) and hit the button
-labeled `New API key`. This will generate a set of keys which can be used with ArangoDB's public API.
 
-### Retrieving relevant IDs
+You need to supply proper credentials to the provider before it can be used. API keys serve as the credentials to the provider. You can obtain the keys from the Oasis dashboard.
+Log in to the Oasis dashboard and open the [API Keys](https://cloud.arangodb.com/dashboard/user/api-keys) tab of your user account. Click the New API key button to generate a new key, which can be used with ArangoDB's public API.
```
Comment (ansoboleva) [inline docs/guides/getting-started.md:16]: should be bold as it is a UI element

File: docs/guides/setup.md [hunk 3]
```diff
@@ -6,9 +6,9 @@ description: |-
 
 # Setup
 
-When using the provider you need to setup `api_key_id` and `api_key_secret`. The two API keys can be generated from Oasis Dashboard. On a logged in view, navigate to [API Keys](https://cloud.arangodb.com/dashboard/user/api-keys)
+When using the provider, you need to set up api_key_id and api_key_secret. Both parameters can be generated in the Oasis Dashboard. Once you are logged in, navigate to the [API Keys](https://cloud.arangodb.com/dashboard/user/api-keys) tab of your user account and click the New API key button.
```
Comment (ansoboleva) [inline docs/guides/setup.md:9]: The button name should be bold as it is a UI element

**Review by ansoboleva — APPROVED** [review]
please address two comments about the formatting, otherwise LGTM

File: docs/guides/getting-started.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ewoutp) [reply docs/guides/getting-started.md:8]: Please remove the word cluster

File: docs/guides/getting-started.md [hunk 4]
```diff
@@ -5,39 +5,64 @@ description: |-
 ---
 
 # Getting started with Oasis Terraform Provider
+ArangoDB Oasis, the ArangoDB Cloud, provides ArangoDB databases as a Service (DBaaS). It enables you to use the entire functionality of an ArangoDB cluster deployment without the need to run or manage the system yourself.
 
+Terraform Provider Oasis is a plugin for Terraform that allows for the full lifecycle management of ArangoDB Cloud resources.
```
Comment (ewoutp) [inline docs/guides/getting-started.md:10]: We need naming. The line before it is called ArangoDB Oasis and here ArangoDB cloud

File: templates/guides/getting-started.md.tmpl [hunk 5]
```diff
@@ -5,39 +5,64 @@ description: |-
 ---
 
 # Getting started with Oasis Terraform Provider
+ArangoDB Oasis, the ArangoDB Cloud, provides ArangoDB databases as a Service (DBaaS). It enables you to use the entire functionality of an ArangoDB cluster deployment without the need to run or manage the system yourself.
 
+Terraform Provider Oasis is a plugin for Terraform that allows for the full lifecycle management of ArangoDB Cloud resources.
```
Comment (ewoutp) [inline templates/guides/getting-started.md.tmpl:10]: Same as before

**Review by ewoutp — CHANGES_REQUESTED** [review]

**Pull request merged**